src/Plugin.php

```diff
@@ -124,6 +124,13 @@ class Plugin extends CommonDBTM
      */
     private static $loaded_plugins = [];
 
+    /**
+     * Store additional infos for each plugins
+     *
+     * @var array
+     */
+    private ?array $plugins_informations = null;
```

Comment:
```suggestion
    private ?array $plugins_information = null;
```

"information" is uncountable

Reply:
Done.